Quiz/quiz.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./quiz.css">
+    <title>Quiz</title>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+           <a href="../index.html"> <h3>Quiz-Miniac</h3></a>
+        </div>
+        <div class="main-container">
+            <div class="cont-subhead">
+                F.R.I.E.N.D.S
+            </div>
+            <div class="question-cont">
+                <div class="question">
+                    <span>Q1. How hours of Friends episodes are there to watch?</span>
+                </div>
```

Comment:
wrapping a span in a div is kind of not needed, you can either use div only or span only, both not needed

Reply:
Ok